core/invoker/src/main/scala/org/apache/openwhisk/core/containerpool/docker/DockerContainerFactory.scala

```diff
@@ -141,3 +141,55 @@ object DockerContainerFactoryProvider extends ContainerFactoryProvider {
   }
 
 }
+
+/**
+ * This contains factory provides a Docker for Mac client which exposes action container's ports on the host.
+ */
+object DockerForMacContainerFactoryProvider extends ContainerFactoryProvider {
```

Comment:
Better to move this class in independent file say `DockerForMacClient.scala`

Reply:
done. Created `DockerForMacContainerFactory.scala`